test/TestExpectations.json

```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "testIdPattern": "[click.spec] Page.click should click the button with fixed position inside an iframe",
```

Comment:
does it have to remain skipped for chrome? what is the reason?

Reply:
Yes, see the changes from [test/src/click.spec.ts](https://github.com/puppeteer/puppeteer/pull/9642/files#diff-b088646a6db20971154462a015176048ac320186b35b21dce04eac1767c9d0b9). It was skipped within the test itself before.

Reply:
Happy to revert if you think that it shouldn't be changed at the moment.

Reply:
Thanks, I overlooked that it was skipped